### DISCUSSION TITLE
Im Gonna Be Setting up a linux mint test vm!

### DISCUSSION THREAD

**Discussion by Rubiidev-18** [cytrinc/bark-coding#88] [Admin General]
if qwerty **allows me** to do this, ill be testing issues using mint!

edit: already setting up the vm rn

edit 2: typing this on the vm, i have set it up!

**Comment by raynepaws** [comment]
what do ya need my permission for?

**Reply by rubiidev18alt** [reply]
eh idrk, if your not on the job and want me to use it

**Reply by Rubiidev-18** [reply]
what my alt said
